ACP2E-338: [Documentation] Add notice that mini cart may be not synchronized across the multiple devices and browsers (#1624)
magento/merchdocs: src/sales/mini-cart.md

![The shopper displays the shopping cart sidebar from a product page]({% link sales/assets/storefront-mini-cart-watch.png %}){: .zoom}
_Mini Cart_

{:.bs-callout-info}

> In some cases _registered_ customer Mini Cart may be not synchronized across the multiple devices and browsers automatically. To synchronize your Mini Cart in such cases, just open the [Shopping Cart]({% link sales/cart.md %}) page on that another device or browser.

Reviewer: ```suggestion
For a _registered_ customer, there are cases when the Mini Cart may not be synchronized across the multiple devices and browsers automatically. To synchronize the Mini Cart in such cases, the customer can simply open the [Shopping Cart]({% link sales/cart.md %}) page on that device or browser.
```

Author: Suggested changes were committed
